### ISSUE TITLE
Missing Staten Island Railway 

### ISSUE THREAD

**Issue by gtbot2007** [BenMusch/nycguessr#3]
The Staten Island Railway (and JFK AirTrain but who cares lol) which are on the subway map

**Comment by BenMusch** [comment]
Yep, wasn't in the dataset unforunately. Looking into adding it and can use this issue to track!
